apps/shell/README.md

```diff
@@ -0,0 +1,80 @@
+# TASH - TinyAra SHell
+
+lightweight shell
+
+## How to enable
+
+Enable configuration of TASH through menuconfig
+```bash
+cd $TIZENRT_BASEDIR
+cd os
+make menuconfig
+```
+Select menu
+```bash
+Application Configuration -> Shell -> Enable shell to y
+```
+## How to register a command on TASH
+
+There are two methods, in code and in Makefile
```

Comment:
A command can be registered in TASH using two methods, one through calling public API's and second through adding the commands in makefile.